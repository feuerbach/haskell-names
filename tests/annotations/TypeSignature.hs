module TypeSignature where

f,g :: Int -> Int
f x = 2 * x
g x = x + x

