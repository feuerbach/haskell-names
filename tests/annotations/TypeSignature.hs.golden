f        at  3:1 is a global value, TypeSignature.f
g        at  3:3 is a global value, TypeSignature.g
Int      at  3:8 is not in scope
Int      at  3:8 is not in scope
Int      at 3:15 is not in scope
Int      at 3:15 is not in scope
f        at  4:1 is a value bound here
x        at  4:3 is a value bound here
*        at  4:9 is not in scope
*        at  4:9 is not in scope
x        at 4:11 is a local value defined at 4:3
x        at 4:11 is a local value defined at 4:3
g        at  5:1 is a value bound here
x        at  5:3 is a value bound here
x        at  5:7 is a local value defined at 5:3
x        at  5:7 is a local value defined at 5:3
+        at  5:9 is not in scope
+        at  5:9 is not in scope
x        at 5:11 is a local value defined at 5:3
x        at 5:11 is a local value defined at 5:3
